office-backend/package.json

```
@@ -9,15 +9,16 @@
   },
   "keywords": [],
   "author": "",
-  "license": "EUPL-1.2+",
+  "license": "ISC",
```

```suggestion
  "license": "EUPL-1.2+",
```
Please revert this change !